crowdgezwitscher/events/tests/test_api_views.py

```diff
@@ -297,6 +218,50 @@ def test_json_detail_events_incorrect(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
+    # GET /api/events/1/tweets
+    def test_get_tweets(self):
+        url = reverse('events_api:tweets', kwargs={'pk': 1})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # As this view combines quite some data, this tests needs some documenentation.
+        # Please update it if you change the fixtures.
+        #
+        # The event is limited to the hashtags "wow" and "awesome".
+        # Only tweets from 2016-07-24 until 2016-07-25 are considered.
+        # The event is limited to the twitter accounts "streetcoverage", "inglor.basterds" and "stalingrad42".
+        # There are 3 tweets, each from one of the accounts with at least one of the specified hashtags and within the
+        # creation date range:
+        response_json = ['11', '22', '66']
+        self.assertEqual(json.loads(response.content.decode("utf-8")).sort(), response_json.sort())
```

Comment:
@piepmatz: Hab am APIView gebastelt und mich gewundert, dass der Test nicht gefailed ist trotz meiner Umbauten. In Python gibt es `sorted(list)` und `list.sort()`. Letzteres wird hier verwendet, arbeitet in-place und gibt `none` zurück, somit wird nur `none == none` geprüft. Ersteres hingegen gibt eine sortierte Kopie von `list` zurück. Fix folgt umgehend.